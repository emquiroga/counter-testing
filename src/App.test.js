import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

test('Render the App', () => {
  render(<App />);
  expect(screen.getByRole("app")).toMatchSnapshot();
});
