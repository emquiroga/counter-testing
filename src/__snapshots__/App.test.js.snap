// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the App 1`] = `
<div
  role="app"
>
  <h2
    role="counter"
  >
    Counter: 
    0
     
  </h2>
  <button
    aria-label="add"
  >
    1
  </button>
  <button
    aria-label="reset"
  >
    Reset
  </button>
  <button
    aria-label="rest"
  >
    -1
  </button>
</div>
`;
